spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -292,7 +294,20 @@ public SparkContext createSparkContext() {
     }
     if (System.getenv("SPARK_HOME") != null) {
       conf.setSparkHome(System.getenv("SPARK_HOME"));
+
+      // load configs from spark properties file
+      final String sparkPropertiesFile =
+          Utils.getDefaultPropertiesFile(JavaConversions.asScalaMap(System.getenv()));
```

Comment:
Or it could return null
https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/util/Utils.scala#L1848